extension/manifest.json

```diff
@@ -4,11 +4,10 @@
   "applications": {
     "gecko": {
       "id": "proxy-failover@mozilla.com",
-      "strict_min_version": "78.0",
-      "strict_max_version": "100.0"
+      "strict_min_version": "78.0"
```

Comment:
@willdurand This version is now strict_min_version >= 91 (I recall that at least one of the globals was added in the sandbox where the api.js script is executed, `WebExtensionPolicy` if I'm not mistaken, was added in 91)

If we still need to ship this into Firefox version < 91, then we should add back globals definition for the stuff that wasn't yet available in the WebExtensions API scripts global in those Firefox versions.